velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -235,23 +337,67 @@ void WindowFuzzer::go() {
     const auto call =
         makeFunctionCall(signature.name, argNames, false, false, ignoreNulls);
 
+    auto tup = frameWindowTypeAndBoundType();
+    auto frameType = std::get<0>(tup);
+    auto startBoundType = std::get<1>(tup);
+    auto endBoundType = std::get<2>(tup);
+
+    std::vector<core::WindowNode::BoundType> kBoundOptions = {
+        core::WindowNode::BoundType::kPreceding,
+        core::WindowNode::BoundType::kFollowing};
+    bool isKRangeFrame = frameType == core::WindowNode::WindowType::kRange &&
+        (std::find(
+             kBoundOptions.begin(), kBoundOptions.end(), startBoundType) !=
+             kBoundOptions.end() ||
+         std::find(kBoundOptions.begin(), kBoundOptions.end(), endBoundType) !=
+             kBoundOptions.end());
+
     std::vector<SortingKeyAndOrder> sortingKeysAndOrders;
+    TypeKind orderByTypeKind;
+    std::string orderByKey = "";
     // 50% chance without order-by clause.
-    if (vectorFuzzer_.coinToss(0.5)) {
+    if (!isKRangeFrame && vectorFuzzer_.coinToss(0.5)) {
       sortingKeysAndOrders =
           generateSortingKeysAndOrders("s", argNames, argTypes);
     }
+    // k-range frames need 1 order by key
+    else {
+      if (requireSortedInput) {
+        sortingKeysAndOrders =
+            generateSortingKeysAndOrders("s", argNames, argTypes, 0);
```

Comment:
`maxNumKeys` cannot be 0 for k Range frames, could you please recheck this conditional @minhancao ?